locals.tf

```diff
@@ -152,6 +152,35 @@ locals {
     "udp",
   ]
 
+  # Ports to be accessed via the VPN in assessment environments
+  # (e.g. for Guacamole, Mattermost, etc.)
+  assessment_env_service_ports = {
```

Comment:
So while re-reviewing I noticed that `ipa_ports` (above) uses `proto` but this map uses `protocol`. I think we should standardize against one or the other to keep things consistent (`ipa_ports` also doesn't have sorted keys but that's a thing for another time).

Reply:
See commit e0cd6a2.